core/src/main/resources/org/apache/struts2/default.properties

```diff
@@ -69,6 +69,7 @@ struts.multipart.parser=jakarta
 struts.multipart.saveDir=
 struts.multipart.maxSize=2097152
 struts.multipart.maxFiles=256
+struts.multipart.maxFileSize=2097152
```

Comment:
We could always make this `-1` so it's unused by default and won't break backwards compatibility for existing Struts applications.

Reply:
Makes sense, and also this option somehow duplicates what we have on the interceptor level - so this is a global config, which takes precedence over the interceptor configuration.

I would just comment out this line and use `null` as a indicator to mark this option disabled to match the same behaviour for other options.

Reply:
Done :)